cluster/juju/layers/kubernetes-master/layer.yaml

```diff
@@ -9,10 +9,12 @@ includes:
   - 'interface:ceph-admin'
   - 'interface:public-address'
 options:
+  basic:
+    packages:
+      - 'nfs-common'
```

Comment:
Kubernetes master is not loading volumes, so it seems unnecessary to load this module here.